build.gradle

```diff
@@ -47,9 +47,10 @@ plugins {
   // Updating the shadow plugin version to 8.1.1 causes issue with signing and publishing the shadowed
   // artifacts - see https://github.com/johnrengelman/shadow/issues/901
   id 'com.github.johnrengelman.shadow' version '8.1.0' apply false
-  //  the minimum required JRE of 6.14.0+ is 11
```

Comment:
Could you please add more comments to explain why we exclude JDK 8?  The previous discussion can be referenced

https://github.com/apache/kafka/pull/16172#issuecomment-2145167748